client/my-sites/sidebar-unified/style.scss

```diff
@@ -72,6 +72,20 @@ $font-size: rem( 14px );
 				background-color: var( --color-sidebar-menu-hover-heading-background );
 				color: var( --color-sidebar-menu-hover );
 			}
+
+			&::after {
+				display: none;
```

Comment:
Minor: The property order is a bit all over the place here (see [core coding style guidelines](https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#property-ordering)). I know this is copy pasted from an existing place but maybe we could improve it while we're at it. It's optional but maybe we could at least group the positioning related items together? That'll make it easier to grasp when scanning/maintaining the code.

Reply:
Yeah, good chance to revisit the CSS code standards! Fixed 94e979a